cmd/snappy/cmd_install.go

```diff
@@ -76,7 +79,12 @@ func (x *cmdInstall) doInstall() error {
 	// TRANSLATORS: the %s is a pkgname
 	fmt.Printf(i18n.G("Installing %s\n"), pkgName)
 
-	realPkgName, err := snappy.Install(pkgName, flags, progress.MakeProgressBar())
+	channel := release.Get().Channel
+	if idx := strings.IndexByte(pkgName, '/'); !helpers.FileExists(pkgName) && idx > -1 {
```

Comment:
It ought to check the index first. No need to touch the filesystem if it doesn't look like the right kind of data.

That said, we agreed at the sprint to use a proper -c flag for this instead of mangling them together. Is there any reason to not do that now?